src/c++/perf_analyzer/client_backend/client_backend.h

```diff
@@ -144,6 +144,10 @@ struct ModelStatistics {
   uint64_t success_count_;
   uint64_t inference_count_;
   uint64_t execution_count_;
+  uint64_t queue_count_;
```

Comment:
Does it make sense to use std::pair to hold the `*_ns_` and `*_count_` ?

Reply:
Punting on this for now, if that's alright--can do in separate PR 👍